tests/unit/utils/console/test_logger_results.py

```
@@ -99,13 +103,15 @@ def test_print_tool_result_handles_pytest_for_test_action(
+    with (
+        patch.object(logger, "console_output"),
+        patch.object(logger, "_print_pytest_results") as mock_pytest,
+    ):
+        logger.print_tool_result(
+            "pytest",
+            "test output",
+            0,
+            action=Action.TEST,
+            success=True,
+        )
+        mock_pytest.assert_called_once_with("test output", True)
```

Ruff (0.15.6)

[warning] Boolean positional value in function call

(FBT003)